addons/account/models/account_move.py

```diff
@@ -5710,7 +5716,7 @@ def _autopost_draft_entries(self):
                         move._post()
                 except UserError as e:
                     move.checked = False
-                    msg = _('The move could not be posted for the following reason: %(error_message)s', error_message=e)
+                    msg = _('Please provide a customer to validate the invoice.\n %(error_message)s', error_message=e)
```

Comment:
The message here is generic.
Are you sure it will always be because there is a missing customer?